### PR TITLE
WAR-1296:update netconf related files

### PR DESCRIPTION
**Pull request by YutakaMizugaki**
WAR-1296: supports netconf protocol version 1.1
and add get-schema and cancel-commit
WAR-1137: add clear notification buffer
WAR-1138: add print notification buffer

### CODE REVIEW

File: warrior/Actions/NetconfActions/netconf_Actions.py [hunk 1]
```diff
@@ -612,18 +690,25 @@ def edit_config_from_string(self, datastore, config, system_name,
...
-        netconf_object = Utils.data_Utils.get_object_from_datarepository(session_id)
+        netconf_object = Utils.data_Utils.get_object_from_datarepository(
+            session_id)
 
         reply = netconf_object.edit_config(datastore, config,
                                            default_operation=default_operation,
                                            test_option=test_option,
                                            error_option=error_option)
-        pNote('Edit Config Reply= {}'.format(reply))
+        # pNote('Edit Config Reply= {}'.format(reply))
```
Comment (navvrnd) [inline warrior/Actions/NetconfActions/netconf_Actions.py]: please remove the commented line, if not needed

File: warrior/Actions/NetconfActions/netconf_Actions.py [hunk 2]
```diff
@@ -793,3 +884,155 @@ def testfor_killsession(self, system_name, session_name=None):
...
+        """
+        wdesc = "clear notification buffer all"
+        pSubStep(wdesc)
+        pNote(system_name)
+        pNote(self.datafile)
+        session_id = Utils.data_Utils.get_session_id(system_name, session_name)
+        temp_dict = Utils.config_Utils.data_repository
+        for s0, s1 in temp_dict.items():
+            if s0 != session_id and isinstance(s1, WNetConf):
+                #pNote("clear for %s" %s0)
```
Comment (navvrnd) [inline warrior/Actions/NetconfActions/netconf_Actions.py]: remove commented line if not needed

File: warrior/Framework/ClassUtils/netconf.py [hunk 3]
```diff
@@ -66,39 +71,45 @@ def __init__(self):
         self.__wait_rept = Event()
         self.__wait_rept.clear()
         self.__response_buffer = ""
-        self.__notification_buffer = ""
+        # self.__notification_buffer = ""
```
Comment (navvrnd) [inline warrior/Framework/ClassUtils/netconf.py]: remove commented line if not needed

**Comment by YutakaMizugaki** [comment]
Thank you for your comments.
I had removed the commented lines per review comments.

**Pull request merged**